src/main/java/io/jenkins/plugins/audit/RequestContext.java

```diff
@@ -39,7 +39,7 @@
     @ClientServer
     public static final String REQUEST_URI = "requestUri";
     @ClientServer
-    public static final String NODE_NAME = "nodeName";
+    public static final String NODE_NAME = "master";
```

Comment:
This should be the attribute name, not the attribute value. We'll be setting the value to `master` by default, though.

Reply:
oh I already changed that in the last commit